Dockerfile-navigator

```diff
@@ -6,11 +6,13 @@
 ARG sdk_version=0.13.22
 
 FROM digitalasset/daml-sdk:${sdk_version}
-WORKDIR /home/sdk
-COPY .daml/dist/CdmSwaps-1.0.0.dar /home/sdk/
-COPY ui-backend.conf frontend-config.js /home/sdk/
 
 EXPOSE 4000
```

Comment:
OK, let's move this line back down to just before the `CMD` so we can share the cache with _Dockerfile-sandbox_.

Reply:
think it is leveraging the caching now